src/sonic-py-common/sonic_py_common/sonic_interface.py

```diff
@@ -0,0 +1,45 @@
+import sys
+
+"""
+ Dictionary of SONIC interface name prefixes. Each entry in the format
+ "Human readable interface string":"Sonic interface prefix"
+ Currently this is a static mapping, but in future could be stored as metadata in DB.
+"""
+
+SONIC_INTERFACE_PREFIXES = {
+  "Ethernet-FrontPanel":"Ethernet",
+  "PortChannel":"PortChannel",
+  "Vlan":"Vlan",
+  "Loopback":"Loopback",
+  "Ethernet-Backplane":"Ethernet-BP"
+}
+
+def sonic_fp_interface_prefix():
```

Comment:
Suggest removing the `sonic_` prefix from the functions and the filename, since the package is already called `sonic-py-common`

Reply:
Ok, will remove the sonic_ prefix from functions. For the file name I thought would be good to import as "from sonic_py_common import sonic_interface" and use it as sonic_interface.functions(). Let me know - but I get you point as well as it is in sonic-py-common already.

Reply:
I see your point, as well. But I think making the name shorter is more convenient. However, I can see that calling `interface.function()` is a bit vague, but one only needs to look to the imports to realize the `interface` module was imported from `sonic-py-common`.